icons/square/48/telegram.svg

```diff
@@ -1,33 +1,15 @@
-<svg width="48" height="48" version="1.1" viewBox="0 0 48 48.000001" xmlns="http://www.w3.org/2000/svg">
+<svg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you please readd `width="48" height="48"` attributes to the `svg` declaration? That should fix the web preview.

Reply:
GitHub's SVG preview is notoriously temperamental so it would be a fools errand to try and keep all our icons compatible with its bugs and quirks (c.f. the SVG diff preview has been broken for several months now despite no changes anywhere else). Browsers themselves can render the icons fine as which is all that matters from a web perspective, and even then that's only so that browser-based applications can render them fine.